include/solvers/navier_stokes_scratch_data.h

```diff
@@ -1364,6 +1373,13 @@ class NavierStokesScratchData : public PhysicsScratchDataBase
         this->fe_values_particle_velocity->reinit(particle_velocity_cell);
```

Comment:
I would move the comment on the implicit and semi-implicit coupling (currently lines 1364-1366) in the `else` scope

_// particle_drag_values will remain zero in the implicit and
 // semi-implicit coupling, since the momentum transfer
// coefficient is used instead_